packages/react-native/src/evm/providers/thirdweb-provider.tsx

```diff
@@ -79,7 +80,12 @@ export const ThirdwebProvider = <
     <ThirdwebProviderCore
       supportedWallets={supportedWallets}
       storageInterface={
-        storageInterface || new ThirdwebStorage({ clientId: clientId })
+        storageInterface ||
+        new ThirdwebStorage({
+          clientId: clientId,
+          /// @ts-ignore
+          gatewayUrls: sdkOptions?.supportedChains,
```

Comment:
this seems wrong

Reply:
jeez, I was testing other props to see if typescript recognized them and left this :(